### ISSUE TITLE
Quest Emergency Measures - NPC's not respawning as expected

### ISSUE THREAD

**Issue by Auviena** [dalaranwow/dalaran-wow#5020]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
For the quest Emergency Measures (QuestID 13000) the player is asked to collect 6 Worg Fur's (ItemID 42510) which drops from Cavedweller Worg (NPCID = 30164) inside a cave in Storm Peaks at approximatly 49,52.

These Worgs seem not respawning at all. There's corpses of them all through the cave, but no live ones.
I went to the cave prior an announced server-restart and logged back in as soon as the server was back online, only to find the worg-corpses still being there.
I also killed other mobs in the cave and waited for their respawn to see if worgs were to spawn in between, but they did not.

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Thank you very much for your report. After the next maintenance which shall be within 2 hours as much, this issue will be corrected.
